frontend/src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const proxy = require('http-proxy-middleware');
+
+module.exports = function(app) {
+  app.use(proxy('/query', { target: 'http://localhost:8080' }));
```

Comment:
we are not limited by json anymore. Let's make it a bit prettier:
```js
['/query',  '/export', ...].forEach(path => app.use(proxy(path, { target: 'http://localhost:8080' })))
```

Reply:
:+1: done in 8a440da.